src/tracker/index.js

```diff
@@ -16,7 +16,7 @@ const UPDATE_DOCUMENT_LABEL = config.get('tracker.githubIssues.label') || 'updat
 const LOCAL_CONTRIBUTE_URL = 'http://localhost:3000/en/service';
 const CONTRIBUTE_URL = 'https://contribute.opentermsarchive.org/en/service';
 const GITHUB_VERSIONS_URL = 'https://github.com/ambanum/OpenTermsArchive-versions/blob/master';
```

Comment:
I think this should also be in a config variable as it differs from project to project

Reply:
Ok, which name can we give to this variable?

Reply:
This certainly has to change. Right now the information provided to the user is misleading.

Considering the complexity of naming and defining this variable vs the uncertainty of the added value (it is used only on line 238 to give a direct link to the versions generated by the failing service declaration —this is certainly useful, but I just don't know how much right now), I'd be in favour of dropping this altogether.

If someone comes up with a great naming idea, let's go with it of course 🙂